generators/utils.js

```diff
@@ -51,21 +51,6 @@ const updateEntityAudit = function (entityName, entityData, javaDir, resourceDir
         this.replaceContent(`${javaDir}repository/${entityName}Repository.java`, `public interface ${entityName}Repository`, `@JaversSpringDataAuditable\npublic interface ${entityName}Repository`);
         this.replaceContent(`${javaDir}repository/${entityName}Repository.java`, `domain.${entityName};`, `domain.${entityName};\nimport org.javers.spring.annotation.JaversSpringDataAuditable;`);
       }
-      // update the list of audited entities if audit page is available
-      if (updateIndex && this.fs.exists(`${javaDir}web/rest/JaversEntityAuditResource.java`)) {
-        this.existingEntities.push(entityName);
-        this.auditedEntities = [];
-
-        this.existingEntities.forEach((entityName) => {
-          this.auditedEntities.push(`'${entityName}'`);
-        });
-
-        const files = [{
-          from: `${this.javaTemplateDir}/web/rest/_JaversEntityAuditResource.java`,
```

Comment:
Yeah, it is, sorry, I corrected myself, didn't notice that the `:entity` subgenerator uses this. I'll submit a new PR.